places-solution-integrations/analytics/places-trigger.md

```diff
@@ -0,0 +1,68 @@
+# Using Places Events to Send Data to Analytics via Launch Rules
+
+This document assumes that you have Adobe Places implemented in your application. If you need help implementing Adobe Places, see [Places Extensions for AEP SDK](../../configure-places-in-the-sdk/README.md).
```

Comment:
- Make this an "important" note. 
- Change the 2nd sentence to read: 
"For more information about implementing Places..."